cfd_protocol/tests/cfds.rs

```diff
@@ -29,32 +29,41 @@ fn create_cfd() {
     let taker_wallet = build_wallet(&mut rng, Amount::from_btc(0.4).unwrap(), 5).unwrap();
 
     let oracle = Oracle::new(&mut rng);
-    let (nonce_commitment, announcement) = announce(&mut rng);
+    let (nonce_commitment_0, announcement_0) = announce(&mut rng);
+    let (nonce_commitment_1, announcement_1) = announce(&mut rng);
+
+    let mut payouts_per_event = HashMap::new();
+    let nonce_pks_0 = announcement_0.nonce_pks();
 
-    let payouts = vec![
+    let event_0 = Event {
+        id: "id".to_string(),
+        nonce_pks: nonce_pks_0,
+    };
+    let nonce_pks_1 = announcement_1.nonce_pks();
+    let event_1 = Event {
+        id: "id".to_string(),
```

Comment:
Why are these IDs not different?

Reply:
Good catch 🤦‍♂️

Because of this we ended up not having any CETs in this test.